### DISCUSSION TITLE
RaspBerry PI 5 - Rustdesk Not Displaying Correctly

### DISCUSSION THREAD

**Discussion by sdhandyjr** [rustdesk/rustdesk#7800] [Q&A]
Good Day All,

   I have an issue. I have installed Rustdesk client on a Raspberry PI 5 device running Bookworm x64.  I changed my display to use x11 display and yet I get a garbled video display.

Is this a bug? Can someone assist?

Thanks
sdhandyjr

![Screenshot 2024-04-22 232145](https://github.com/rustdesk/rustdesk/assets/162525789/8aa23229-640b-4047-8277-112b35701302)

**Comment by SoyDiego** [comment]
I have the same problem! any solution?

Thanks

**Comment by gdesanto** [comment]
I'm having trouble just getting it installed on my rpi 5 (running bookworm).  Which package / method did you use to install it?

**Comment by xlionjuan** [comment]
No problems on Ubuntu 24.04/25.04/25.10 with RPI5
